scripts/dbMaintenance/removeOrphanFiles.py

```diff
@@ -1,9 +1,12 @@
 import sys, os
 
-import gridfs
 from pymongo import MongoClient
+import gridfs
 import re
-import StringIO
+try:
```

Comment:
instead of try/except, probably better to just use an if:
```suggestion
if sys.version_info >= (3, 0):
   from StringIO import StringIO
else:
    from io import BytesIO
```